app/easyaccess.py

```diff
@@ -150,15 +150,54 @@ def main():
 
     # CONVERT TO ACCESS AND WEB FORMATS
     try:
-        logging.info("Converting to access format...")
-        access_metadata = convert_and_get_metadata(source_file_path, access_file_path, settings.ACCESS_FFMPEG_ARGS, vernon_id, access_file_type, title)
-        logging.info("Converting to access format... DONE\n")
-        if settings.TRANSCODE_WEB_COPY:
-            logging.info("Converting to web format...")
-            web_metadata = convert_and_get_metadata(source_file_path, web_file_path, settings.WEB_FFMPEG_ARGS, vernon_id, web_file_type, title)
-            logging.info("Converting to web format... DONE\n")
+        if settings.EXHIBITIONS_TRANSCODER:
+            # Transcoder settings for in-gallery exhibitions videos
+            logging.info('Converting to exhibitions access format...')
+            access_metadata = convert_and_get_metadata(
+                source_file_path,
+                access_file_path,
+                settings.EXHIBITIONS_ACCESS_FFMPEG_ARGS,
+                vernon_id,
+                access_file_type,
+                title,
+            )
+            logging.info('Converting to access format... DONE\n')
```

Comment:
```suggestion
            logging.info('Converting to exhibitions access format... DONE\n')
```

Reply:
@alihaberfield totally agree. Probably good to do this as part of #20